src/main/java/com/fontys/api/service/TournamentService.java

```diff
@@ -24,14 +25,37 @@ public TournamentService(TournamentRepository tournamentRepository, UserReposito
 
     @Transactional
     public Tournament createTournament(String name, String description, Integer ownerId, int numberOfTeams)
+    throws InvalidAttributeValueException
     {
-        User user = userRepository.findById(ownerId).get();
+        if (numberOfTeams <= 1)
+        {
+            throw new InvalidAttributeValueException(String.format(
+                    "A tournament must be created for at least 2 teams. Number of teams provided was %d." +
+                    " Please change the value and try again.", numberOfTeams));
+        }
+        if (name == null || name.isBlank())
+        {
+            throw new InvalidAttributeValueException(
+                    "The tournament name can't be empty. Please give your tournament a name and try again.");
+        }
+        if (ownerId == null || ownerId < 1)
+        {
+            throw new InvalidAttributeValueException(
+                    "Something went wrong while creating the tournament. Please try again.");
+        }
+        User user = userRepository.findById(ownerId).orElse(null);
```

Comment:
This would allow us to create tournaments without owners (.orElse(null);).
If the 'user' is null, we also should return an exception.